src/masoniteorm/scopes/SoftDeletesMixin.py

```diff
@@ -4,5 +4,10 @@
 class SoftDeletesMixin:
     """Global scope class to add soft deleting to models."""
 
+    __deleted_at__ = "deleted_at"
```

Comment:
This value gets overidden at the model level. Since this Mixin class inherited by the model. We can just pass this value into the scope class. Inside the constructor of the scope we can set it on the scope class itself.

```python
class SoftDeleteScope:

    def __init__(self, deleted_at_column="deleted_at"):
        self.deleted_at_column = deleted_at_column
```